src/Sentry/Internal/ProcessInfo.cs

```diff
@@ -100,4 +119,32 @@ private static DateTimeOffset GetStartupTime()
         return proc.StartTime.ToUniversalTime();
     }
 #endif
+
+    public bool? ApplicationIsActivated(SentryOptions options)
+    {
+        try
+        {
+            var activatedHandle = GetForegroundWindow();
```

Comment:
Will crash on mac/linux, right?

Reply:
Yes, but it is called only on Windows but I'll move OS check inside this method.